src/components/Blog/Blog.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import BlogDetails from "./BlogDetails";
+import { Container, Row, Col, Spinner } from "react-bootstrap";
+
+const Blog = () => {
+  const [allPosts, setAllPosts] = useState([]);
+
+  const getAllPosts = async () => {
+    const data = await fetch(
+      "https://public-api.wordpress.com/wp/v2/sites/ecommerce458750409.wordpress.com/posts"
+    );
+
+    const posts = await data.json();
+    console.log(posts);
+    setAllPosts(posts);
+  };
```

Comment:
missing new line

Reply:
hi Halit, i added an extra line as a space. i m sorry today is really busy and not a good day at work so my brain stopped by working since the morning. and i hope i understood your request correctly bcoz dont wanna delay it. please correct me if i m wrong.